internal/generator/vector/output/elasticsearch/elasticsearch.go

```diff
@@ -53,6 +54,7 @@ func New(id string, o obs.OutputSpec, inputs []string, secrets helpers.Secrets,
 			Debug(id, helpers.MakeInputs(inputs...)),
 		}
 	}
+	componentID := helpers.MakeID(id, "elasticsearch_index")
```

Comment:
nit.  you could probably just call this index.  I defer to you if you wish to alter it, but not a blocker for me